modules/multi2vec-dummy/clients/meta.go

```diff
@@ -0,0 +1,20 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package clients
+
+type MetaClient struct{}
+
+func (v MetaClient) MetaInfo() (map[string]interface{}, error) {
```

Comment:
is this `multi2vec-dummy` module finished? It looks like a half-done implementation

Reply:
woops, thanks for noticing!